libethcore/CommonJS.cpp

```diff
@@ -110,11 +110,16 @@ std::string prettyU256(u256 _n)
 	{
 		Address a = right160(_n);
 
-		std::string n = a.abridged();
+		std::string n;
+		if (_abridged)
+			n =  a.abridged();
+		else
+			n = toHex(a.ref());
+
 		if (n.empty())
 			s << "0x" << a;
 		else
-			s << n << "(0x" << a.abridged() << ")";
+			s << n << "(0x" << n << ")";
```

Comment:
Does not make much sense to display the same value twice. I'd suggest displaying 0x0 for zero address, hex value otherwise